x-pack/plugins/security_solution/common/detection_engine/schemas/request/create_rules_type_dependents.ts

```diff
@@ -92,6 +92,19 @@ export const validateTimelineTitle = (rule: CreateRulesSchema): string[] => {
   return [];
 };
 
+export const validateThreshold = (rule: CreateRulesSchema): string[] => {
```

Comment:
The same function is being used in all *dependents.ts files. I am not sure if its best to move it to a common file and being imported from there. I understand that `validateTimelineTitle` etc use the same pattern and maybe it's better to have isolated duplicate code than tightly coupled code. What do you think is best to do in this kind of situation?